src/components/utils/http.js

```diff
@@ -22,7 +22,8 @@ const errorMessage = (status) => {
   }
 }
 
-const handleError = (status, callback) => {
+const handleError = ({status, response}, callback) => {
+  console.error(status, response)
   const message = errorMessage(status)
   Tracker.sendError(message)
```

Comment:
`errorMessage()` is a function that returns a different message based on the status code. The response for 500 errors is "Something went wrong" while for 0 status code, when there is an error in firing the request we don't get a response at all, that's why I'm handling it here.
```const errorMessage = (status) => {
  if (status === 401 || status === 403) return 'unauthorized'
  if (status >= 500) {
    'server error' }
  else {
    'request error'
  }
}```